src/App/ImportUshahidiV2/ImportMappingRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Ushahidi Export Batch Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\ImportUshahidiV2;
+
+use Ushahidi\Core\Entity\ImportMapping;
+
+class ImportMappingRepository /*extends EloquentRepository*/ implements ImportMappingRepositoryContract
```

Comment:
I'm not really sure where this should live. I was kind hoping to avoid our existing repo infrastructure but that might be more confusing. IDN.